src/query/api/v1/handler/namespace/get.go

```diff
@@ -100,3 +116,19 @@ func (h *GetHandler) Get() (nsproto.Registry, error) {
 
 	return protoRegistry, nil
 }
+
+func nanosToDuration(resp proto.Message) (map[string]interface{}, error) {
+	marshaler := jsonpb.Marshaler{EmitDefaults: true}
```

Comment:
Is it really necessary to go from PB -> JSON -> map -> JSON? I assume this is the "most convenient way to get an iterable of durations". How painful are these responses to process? I don't necessarily feel the need to over-engineer this, but if this is a pattern we're going to see more of, it'd be better to create something like an `ConvertibleDurationIterator` interface, a concrete, per-message implementation, and then the conversion func would take an iterator and return a map.

Thoughts?

Reply:
resolving per other convo; let's revisit if we see a pattern emerging